src/components/KeyboardShortcutRow.jsx

```diff
@@ -38,7 +38,7 @@ function KeyboardShortcutRow({
         <span
           {...css(styles.KeyboardShortcutRow_key)}
           role="img"
-          aria-label={label}
+          aria-label={`${label} `}
```

Comment:
Adding in the space here inserts a space between the screen reader reading the label and then the {action} from the <div> below.  Previously it was reading like:

"Enter keySelect the date in focus"

Basically it was merging the last word of one to the first word of the next and making it hard to understand when read.

Reply:
would adding `:` or similar instead of a space have the same effect?

Reply:
Adding a `:` works great with Safari but in Chrome the punctuation is read so it sounds like:

"Enter key colon Select the date in focus"

I tried a `,` instead and that seems to work nicely and consistently in both browsers.  So I'll change this to a `,` instead of a space

Reply:
awesome!